test/e2e/auth/audit.go

```diff
@@ -120,8 +120,8 @@ var _ = SIGDescribe("Advanced Audit", func() {
 				},
 				[]auditEvent{
 					{
-						v1beta1.LevelRequestResponse,
-						v1beta1.StageResponseComplete,
+						auditv1.LevelRequestResponse,
```

Comment:
Honestly, I'd still like to see both `v1` and `v1beta1` being tested for a while, at least partially.

Reply:
Now both v1 and v1beta1 are test in unit test.

But in e2e test we can test only one version. Other wise we need to reset the --audit-log-version option and restart kube-apiserver. Now the output version is audit.k8s.io/v1 by default.

Reply:
Do we have integration tests using k8s.io/kubernetes/cmd/kube-apiserver/app/testing/testserver.go? That way we could test different flags. Very similar to e2e in fact, with a real kube-apiserver. We have some of those tests in test/integration/master.

Reply:
> We have some of those tests in test/integration/master.

Added an integration test.

Reply:
Do I need to copy all testcases from e2e test into integration test?

Reply:
The test you have now looks pretty good. @soltysh anything else you want to see for each version?